apps/server/src/utils/parser.ts

```diff
@@ -194,7 +194,7 @@ export const parseExcel = (excelData: unknown[][], options?: Partial<ImportMap>)
         event.title = makeString(column, '');
         // if this is a block, we have nothing else to import
         if (event.type === SupportedEvent.Block) {
-          continue;
+          /* empty */
```

Comment:
i think the reason the continue was here was to stop the execution of that block.
That in turn would make sure we dont continue parsing an element after we found out it was a block

Why is this being removed? maybe I am missing something?

If there is a reason to remove the continuewe should remove the conditional entirely